interface-definitions/vpp.xml.in

```diff
@@ -876,30 +875,57 @@
                 <help>Outside NAT IP (source NAT only)</help>
               </properties>
               <children>
-                <leafNode name="address">
+                <tagNode name="pool">
                   <properties>
-                    <help>IP address, subnet, or range</help>
-                    <completionHelp>
-                      <list>masquerade</list>
-                    </completionHelp>
-                    <valueHelp>
-                      <format>ipv4</format>
-                      <description>IPv4 address to match</description>
-                    </valueHelp>
+                    <help>Translation IPv4 pool</help>
                     <valueHelp>
-                      <format>ipv4range</format>
-                      <description>IPv4 address range to match</description>
+                      <format>txt</format>
+                      <description>Name of the pool</description>
                     </valueHelp>
+                    <constraint>
+                      #include <include/constraint/alpha-numeric-hyphen-underscore-dot.xml.i>
+                    </constraint>
+                  </properties>
+                  <children>
+                    <leafNode name="address">
+                      <properties>
+                        <help>IP address or range</help>
+                        <valueHelp>
+                          <format>ipv4</format>
+                          <description>IPv4 address to match</description>
+                        </valueHelp>
+                        <valueHelp>
+                          <format>ipv4range</format>
+                          <description>IPv4 address range to match</description>
+                        </valueHelp>
+                        <constraint>
+                          <validator name="ipv4-address"/>
+                          <validator name="ipv4-range"/>
+                        </constraint>
+                      </properties>
+                    </leafNode>
+                  </children>
+                </tagNode>
+              </children>
+            </node>
+            <node name="limits">
+              <properties>
+                <help>Limit settings</help>
+              </properties>
+              <children>
+                <leafNode name="per-thread-sessions">
+                  <properties>
+                    <help>Maximum number of sessions per thread</help>
                     <valueHelp>
-                      <format>masquerade</format>
```

Comment:
What should I do if I have a connection via DHCP and do not know the address? 
Masquerade should help in cases that are common to clients. We should somehow use `masquerade` i.e if the address is not defined, should be getting from the interface (would be an ideal solution)

for example 
eth0 - WAN
eth1 - LAN

I get the IP address by DHCP or just do not want to care about the address, and this address could be different between lease expirations/reboots.